app/src/test/java/uk/ac/ebi/atlas/experimentpage/markergenes/MarkerGenesDaoIT.java

```diff
@@ -111,4 +111,18 @@ void shouldFetchOnlyInferredCellTypeMarkerGenes() {
         return jdbcTestUtils.fetchKsFromCellGroups(EXPERIMENT_ACCESSION_WITH_MARKER_GENES)
                 .stream().map(Object::toString);
     }
+
+    @Test
+    void shouldGetCellTypesWithMarkerGenes() {
+        var cellTypesWithMarkerGenes = subject.getCellTypesWithMarkerGenes("E-MTAB-5061", "Inferred cell type - ontology labels");
+        assertThat(cellTypesWithMarkerGenes).isNotEmpty();
+
+    }
+
+    @Test
+    void shoulGgetCellTypeMarkerGenes() {
```

Comment:
- Typo mistake. 
- The above test method and current test method names are identical, but we are testing different functionality. Can you elaborate on test method names to improve readability?

Reply:
Yes, I refactored here.